dist/nix/quick-lint-js.nix

```diff
@@ -1,9 +1,8 @@
 # Copyright (C) 2020  Matthew "strager" Glazar
 # See end of file for extended copyright information.
 
-{ pkgs
+{ pkgs ? import <nixpkgs>{}
```

Comment:
What's this change for? If it's not related to Python or Clang or caching, put it in its own commit with an explanation.